File: lab1/control/p2.cpp
<?php
if (isset($_POST['Submit']))

{

    $name = $_REQUEST['fname'];
        if ($name == '')
 
    {
        echo 'Please Enter a valid First Name';
    }
         else
    {
        echo 'Your First Name is ' . $name;
    }
    echo '<br>';

    $name2 = $_REQUEST['lname'];
        if ($name2 == '')
    {
        echo 'Please Enter a valid Last Name';
    } else
    {
        echo 'Your Last Name is ' . $name2;
    }

    echo '<br>';

    $Age = $_REQUEST['age'];

        if ($Age == '')
    {
        echo 'Please Enter your Age ';
    } else
    {
        echo 'Your Age is ' . $Age;
    }

    echo '<br>';
   
    if (
        isset($_POST['designation1']) ||    isset($_POST['designation2']) || isset($_POST['designation3']))
        {
        if (isset($_POST['designation1'])) {
            echo 'You  Select Junior Programmer ';
            echo '<br>';
        }
        if (isset($_POST['designation2'])) {
            echo 'You  Select Senior Programmer';
            echo '<br>';
        }
        if (isset($_POST['designation3'])) {
            echo 'You  Select Project Lead';
            echo '<br>';
        }
    } else {
        echo 'You havent selected anything';
    }

    echo '<br>';

   
    if (
        isset($_POST['PreferredLanguage1']) || isset($_POST['PreferredLanguage2']) ||isset($_POST['PreferredLanguage3'])) {
        if (isset($_POST['PreferredLanguage1'])) {
            echo 'You  Select JAVA ';
            echo '<br>';
        }

        if (isset($_POST['PreferredLanguage2'])) {
            echo 'You  Select PHP';
            echo '<br>';
        }
        if (isset($_POST['PreferredLanguage3'])) {
            echo 'You  Select C++';
            echo '<br>';
        }
    }
    else {
        echo 'You Havent Select a checkbox';
        echo '<br>';
    }


    $Password = $_REQUEST['password'];
    if (strlen($Password) < 5) {
        echo 'Enter a valid password';
    } else
     {
        echo 'password is valid';
    }

}
?>

File: Lab_Task-2/lab2/control/p2.php
<?php
if (isset($_POST['Submit'])) 

{
    $FirsName = $_REQUEST['fname'];
        if (empty($FirsName)) 
    {
        echo 'Please Enter a valid First Name';
    }
         else 
    {
        echo 'Your First Name is ' . $FirsName;
    }
    echo '<br>';

        $LastName = $_REQUEST['lname'];
        if (empty($LastName)) 
    {
        echo 'Please Enter a valid Last Name';
    } else
    {
        echo 'Your Last Name is ' . $LastName;
    }

    echo '<br>';

    $Age = $_REQUEST['Age'];

        if ($Age == '') 
    {
        echo 'Please Enter your Age ';
    } else 
    {
        echo 'Your Age is ' . $Age;
    }

    echo '<br>';
    
    if (
        isset($_POST['Desgination1']) ||    // null kina  isset check kore radio button r error
        isset($_POST['Desgination2']) ||
        isset($_POST['Desgination3'])) 
        {
        if (isset($_POST['Desgination1'])) {
            echo 'Your Have Selected Junior Programmer ';
            echo '<br>';
        }
        if (isset($_POST['Desgination2'])) {
            echo 'Your Have Selected Senior Programmer';
            echo '<br>';
        }
        if (isset($_POST['Desgination3'])) {
            echo 'Your Have Selected Project Lead';
            echo '<br>';
        }
    } else {
        echo 'You Have not selected anything';
    }

    echo '<br>';

    
    if (
        isset($_POST['PreferredLanguage1']) ||
        isset($_POST['PreferredLanguage2']) ||
        isset($_POST['PreferredLanguage3'])) {
        if (isset($_POST['PreferredLanguage1'])) {
            echo 'Your Have Selected JAVA ';
            echo '<br>';
        }

        if (isset($_POST['PreferredLanguage2'])) {
            echo 'Your Have Selected PHP';
            echo '<br>';
        }
        if (isset($_POST['PreferredLanguage3'])) {
            echo 'Your Have Selected C++';
            echo '<br>';
        }
    } 
    else {
        echo 'You Have not Selected a checkbox';
        echo '<br>';
    }

    $Email = $_POST['Email'];
    if (
        empty($Email) ||
        !preg_match(
            '/^([a-z0-9\+_\-]+)(\.[a-z0-9\+_\-]+)*@([a-z0-9\-]+\.)+[a-z]{2,6}$/ix',
            $Email
        )
    ) {
        echo 'Invalid email';
        echo '<br>';
    } else 
    {
        echo 'Your email is ' . $Email . '<br>';
    }

    $Password = $_REQUEST['Password'];
    if (strlen($Password) < 8) {
        echo 'Enter a valid password';
    } else
     {
        echo 'password is valid';
    }



}
?>